notebooks/STIS/drizpac_notebook/STIS_DrizzlePac_Tutorial.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>The execution of the notebook in CI depends on having all dependencies installed properly and automatically via requirements.txt. We need to move all these dependencies from markdown to requirements.txt.</p><p> </p><p>The notebook is quite lengthy, therefore it would be helpful to have a table of contents with internal links to each header for easier navigation.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Hi @Jackie-Brown,
I wanted to let you know that the `stis_cti` and `refstis` are currently not available on PyPI. The instructions in the notebook suggest manually cloning the repository and launching it from the setup.py. However, to ensure successful CI runs, we need to automate this step by adding them to the requirements.txt file, like so:

`refstis @ git+https://github.com/spacetelescope/refstis@master`
`stis_cti @ git+https://github.com/spacetelescope/stis_cti@master`

Reply:
<p>@haticekaratay I added in refstis and stis_cti no problem as well as the dependencies listed from the markdown. I also removed conda environment section from the notebook. However there's still an execution error stating that numpy can't be found, which usually comes from a package conflict. I'm still not certain which package is causing the issue</p>

Reply:
To successfully install the `stis_cti`, `numpy` is required. Despite having numpy listed at the top of the requirements.txt, it seems that the dependencies listed with URL have priority over regular packages. Before installing with `requirements.txt`, I will attempt to install `numpy` with pre-installation script.

Reply:
@Jackie-Brown, The installation issue has been resolved.